packages/nc-gui/components/dashboard/TreeView.vue

```diff
@@ -230,7 +235,16 @@ function openTableCreateDialog() {
           </Transition>
 
           <Transition name="layout" mode="out-in">
-            <MdiClose v-if="searchActive" class="text-lg mx-1 mt-0.5" @click="toggleSearchActive(false)" />
+            <MdiClose
+              v-if="searchActive"
+              class="text-lg mx-1 mt-0.5"
+              @click="
+                () => {
+                  filterQuery = ''
+                  toggleSearchActive(false)
+                }
+              "
```

Comment:
Instead of inlining it, could you create a new function? :D 
Vue-TSC complains a lot about these sort of inline functions.

`@click="onClick"`

Reply:
I will update it then :+1: